scripts/restart_ecs

```diff
@@ -3,6 +3,24 @@
 # Restart ECS service
 #
 
+healthcheck_url="https://${APP_ENV}.easi.cms.gov/api/v1/healthcheck"
```

Comment:
This will work for the dev and impl url, but not prod.

Reply:
Thanks, can fix

Reply:
7390af7